frontend/src/slices/auth/actions.ts

```diff
@@ -1,5 +1,6 @@
 import { createAsyncThunk } from '@reduxjs/toolkit';
 
+import { storage, StorageKey } from '~/libs/packages/storage/storage.js';
```

Comment:
```suggestion
import { StorageKey } from '~/libs/packages/storage/storage.js';
```
all services should be passed through extra arguments